app/src/main/java/com/realityexpander/tasky/agenda_feature/data/common/workers/SyncAgendaWorker.kt

```diff
@@ -22,18 +23,18 @@ import java.util.concurrent.TimeUnit
 class SyncAgendaWorker @AssistedInject constructor(
     @Assisted val context: Context,
     @Assisted val workerParams: WorkerParameters,
-    val agendaRepository: IAgendaRepository
+    val agendaRepository: IAgendaRepository,
+    private val workerNotifications: IWorkerNotifications
 ): CoroutineWorker(context, workerParams) {
 
     override suspend fun doWork(): Result {
 
         logcat { "SyncAgendaWorker.doWork() attemptedRuns: ${workerParams.runAttemptCount}" }
         workerParams.log()
 
-        WorkerNotifications.showNotification(
-            context,
-            WorkerNotifications.createNotification(
-                context,
+        workerNotifications.showNotification(
+            workerNotifications.createNotification(
```

Comment:
This is still here

Reply:
Yes, there's not really a scenario where you'd want to create a worker notification, but don't show it right (or show it using a different notification manager)?
Then just exposing showNotification makes total sense